data/notices.json

```diff
@@ -447,6 +447,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(cli) List stack output change issue.",
+      "issueNumber": 29420,
+      "overview": "Expected behavior while running cdk list on dependent stacks maintaining dependency through addDependency is altered",
```

Comment:
Can you add some more detail here? This should give a client information about the incident. As a client, my first question after reading this would be, "altered how?". I also don't think all the initial detail is necessary. Let's do something like:
`v2.132.0 introduced functionality to the cdk list command to display stack dependencies. This feature introduced a change that alters <add-detail-about-what-is-altered>`

Reply:
Updated, thanks Colin!